pom.xml

```
@@ -6,26 +6,13 @@
 
     <groupId>com.hedvig.graphql.commons</groupId>
     <artifactId>graphql-commons</artifactId>
-    <version>1.0.4</version>
+    <version>2.0</version>
```

Sould we keep the Semantic Versioning pattern?
```suggestion
    <version>2.0.0</version>
```